subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -32,6 +32,8 @@
 
 #define BYTES2US(bytes, phy) (((bytes)<<3)/BIT((phy&0x3)>>1))
 
+/* Advertisement channel minimum payload size */
+#define PDU_AC_PAYLOAD_SIZE_MIN 1
```

Comment:
Was any code supposed to be updated to use this?

Reply:
yes it is used in one of later commits: https://github.com/zephyrproject-rtos/zephyr/blob/dcfde9b3acdd4e3dd9b4e398601ca8a364b88a15/subsys/bluetooth/controller/ll_sw/ull_df.c#L792